TEveEventDisplay/src/TEveMu2eDataInterface.cc

```diff
@@ -571,8 +571,8 @@ namespace mu2e{
               segment.helix().position(fltL,pos);
               CLHEP::Hep3Vector p = GenVector::Hep3Vec(pos);
               CLHEP::Hep3Vector InMu2e = det->toMu2e(p);
-              line->SetPostionAndDirectionFromKalRep((InMu2e.z()));
-              line->SetNextPoint((InMu2e.x()), (InMu2e.y()), (InMu2e.z()));
+              line->SetPostionAndDirectionFromKalRep((InMu2e.z()));              
+              line->SetNextPoint(pointmmTocm(InMu2e.x()), pointmmTocm(InMu2e.y()), pointmmTocm(InMu2e.z()));
```

Comment:
I think that I mentioned this before.  It's out of scope for this review but it will make the code base much easier to maintain and I strongly recommend that you do it at your earliest convenience.   Define a set of  inline free functions, to minimize the repeated work.  One example would be:

SetNextPoint ( TEvePointSet* t, CLHEP::Hep3Vector const& p, double factor=1){
  t->SetNextPoint ( factor*p.x(), factor*p.y(), factor*p.z());
}

There are 14 calls to SetNextPoint in this file alone.  Even better, template it on the first argument.  That way it will work with any class that has a member function with the signature SetNextPoint(double, double,double) .

Reply:
True, I will get to this. At the moment I'm focussing on getting REve upto standard, but TEve isnt perfect, I know, due to the way it has evolved

Reply:
> True, I will get to this. At the moment I'm focussing on getting REve upto standard, but TEve isnt perfect, I know, due to the way it has evolved

We will require this sort of change in the REve display so please do it proactively.